services/core-web/common/utils/Validate.js

```diff
@@ -63,6 +65,10 @@ class Validator {
   checkProtocol(url) {
     return this.PROTOCOL_REGEX.test(url);
   }
+
+  checkInteger(url) {
+    return this.INTEGER_REGEX.test(url);
```

Comment:
Are negative values allowed? If not, we should rename it to "checkWholeNumber" or something.

Reply:
Yes, I changed the name to be specific. Thanks